client/state/directly/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DIRECTLY_INITIALIZE
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const isInitialized = createReducer( false, {
+	[ DIRECTLY_INITIALIZE ]: () => true,
+} );
```

Comment:
Right now, there's no use case for `isInitialized` at all so I'm very comfortable removing it. In that respect, there's also not really a use case for tracking `config` at all — should I remove that as well and if we find a need for it later we can re-introduce it?

And re: `createReducer` being heavy — should the default stance be not to use this helper without solid reasoning (vs. using it always to keep reducer markup consistent)?